sdk/storage/azure-storage-blob/azure/storage/blob/_download.py

```diff
@@ -545,7 +566,7 @@ def chunks(self):
 
             iter_downloader = _ChunkDownloader(
                 client=self._clients.blob,
-                non_empty_ranges=self._non_empty_ranges,
+                non_empty_ranges=self._non_empty_ranges,  #type: ignore
```

Comment:
I believe that `_ChunkDownloader` is improperly typed, as it is possible this comes in as `None`? Or maybe it gets initialized previously?

Reply:
azure\storage\blob\_download.py:569: error: Argument "non_empty_ranges" to "_ChunkDownloader" has incompatible type "Optional[List[Dict[str, int]]]"; expected "List[Any]"  [arg-type]